perforce.js

```diff
@@ -51,15 +56,15 @@ function activate() {
 				if(config.addOnFileCreate) {
 					_watcher.onDidCreate(w_onFileCreated);
 				}
-			}
+			}			
 		});
 	}
 }
 exports.activate = activate;
 
 function buildCmdline(command, args)
 {
-	var cmdline = "p4";
+	var cmdline = _cmdline;
 	if (isWin) {
 		cmdline += ".exe";

```

Comment:
Hm, appending '.exe' for Windows will be prone to error. Can we have a 'per-plaform' default in the configuration ?